.eslintrc.json

```diff
@@ -0,0 +1,33 @@
+{
+  "root": true,
+  "parser": "@typescript-eslint/parser",
+  "ignorePatterns": ["src/contracts/*.ts", "src/ecsScene/scene.js", "package.json", "package-lock.json"],
+  "plugins": ["@typescript-eslint"],
+  "extends": ["eslint:recommended", "plugin:@typescript-eslint/recommended", "prettier"],
+  "env": {
+    "browser": true,
+    "es6": true
+  },
+  "rules": {
+    "@typescript-eslint/no-non-null-assertion": "off",
+    "@typescript-eslint/no-unused-vars": "off",
```

Comment:
I would set this as `on` and remove the unused var restriction on the compiler so we can develop without restrictions.